x-pack/plugins/security_solution/common/test/index.ts

```diff
@@ -15,5 +15,3 @@ export enum ROLES {
   platform_engineer = 'platform_engineer',
   detections_admin = 'detections_admin',
 }
-
-export type RolesType = keyof typeof ROLES;
```

Comment:
NOTE: removed as I directly use the ROLES below now and this enables me to use an exhaustive switch.